packages/cli/src/commands/up.ts

```diff
@@ -51,15 +52,20 @@ export default class Up extends MachineCreationDriverCommand<typeof Up> {
       insecureSkipVerify: flags['insecure-skip-verify'],
     }
 
-    const { hostKey, clientId, rootUrl } = await tunnelServerHello({
-      tunnelingKey,
-      knownServerPublicKeys: pStore.knownServerPublicKeys,
-      tunnelOpts,
-      log: this.logger,
-    })
+    const { clientId, rootUrl, hostKey } = await withSpinner(async spinner => {
+      spinner.text = 'Initializing tunnel connection...'
+      return await tunnelServerHello({
+        tunnelingKey,
+        knownServerPublicKeys: pStore.knownServerPublicKeys,
+        tunnelOpts,
+        log: this.logger,
+      })
+    }, { successText: 'Tunnel established.' })
```

Comment:
there is no tunnel being established.

`successText` is optional, can be removed here and it will show a `✔` next to the previous message.

```suggestion
    })
```